drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -128,11 +119,11 @@ class << self
   end
 
   scope :matching_search_term, ->(text_search) do
-    text_searcher(text_search, custom_name_options: CUSTOM_NAME_OPTIONS) do |where|
-      left_outer_joins(:names).where(where).pluck(:id)
-    rescue RangeError
-      return none
-    end
+    text_searcher(text_search, sorted: true)
+  rescue RangeError => e
+    # FIXME: what is this range error?
```

Comment:
I don't know tbh I think it was just copied from the warehouse code. @eanders do you know?

Reply:
Maybe if user searches for a really big number

Reply:
Confirmed that's what it is